rotkehlchen/api/v1/schemas.py

```diff
@@ -453,18 +451,27 @@ def make_trades_query(
             data: dict[str, Any],
             **_kwargs: Any,
     ) -> dict[str, Any]:
+        with self.db.conn.read_ctx() as cursor:
+            treat_eth2_as_eth = self.db.get_settings(cursor).treat_eth2_as_eth
+
         base_assets: Optional[tuple['Asset', ...]] = None
         quote_assets: Optional[tuple['Asset', ...]] = None
         if data['base_asset'] is not None:
             base_assets = (data['base_asset'],)
         if data['quote_asset'] is not None:
             quote_assets = (data['quote_asset'],)
 
-        if self.treat_eth2_as_eth is True and data['base_asset'] == A_ETH:
+        if treat_eth2_as_eth is True and data['base_asset'] == A_ETH:
             base_assets = (A_ETH, A_ETH2)
-        elif self.treat_eth2_as_eth is True and data['quote_asset'] == A_ETH:
+        elif treat_eth2_as_eth is True and data['quote_asset'] == A_ETH:
             quote_assets = (A_ETH, A_ETH2)
 
+        ignored_trades = None
+        if not data['include_ignored_trades']:
```

Comment:
To make the reader sure that the `None` / `False` case is not missed here

```suggestion
        if not data['include_ignored_trades']:  # either False or not specified (None)
```

Reply:
It can only be a boolean in this case, since a default is specified.

Reply:
:+1: Then I would change it to `is False`